### DISCUSSION TITLE
'Done' utilidad

### DISCUSSION THREAD

**Discussion by valeria-adasme** [JavierBZ/Tarea3-IIC3697#17] [Q&A] [answered]
Hola!
al hacer un env.step, una de las salidas es el atributo 'done' que indica si terminé o no el juego (ya sea perdí o gané) ¿Debo realizar alguna decisión en base a eso? Por ejemplo, si hago otro step después de eso sin cambiar nada, ¿el programa entiende que pasé a otro turno o debería haber hecho un reset?

**Comment by JavierBZ** [comment, marked as answer]
Debes usar el done, si revisas el código cuando el done es true se reinicia el ambiente con reset, no lo hace automáticamente.

**Reply by valeria-adasme** [reply]
Muchas gracias, me fijaré entonces en eso!
